README.md

```diff
@@ -343,23 +343,25 @@ module.exports = {
 };
 ```
 
-#### Disable extracting css from async chunks
+#### Disable Extracting Specified CSS from Chunks
 
-You may disable extracting css from async chunks with `disableAsync` option.
+You may disable extracting css from all chunks with `disableExtract` option.
 
 ```javascript
 const miniCssExtractPlugin = new MiniCssExtractPlugin({
-  disableAsync: true,
+  disableExtract: true,
```

Comment:
No need `Boolean` value here, this plugin for extract, and include plugin and disable this is misleading, only `Function` support

Reply:
Boolean type option has been removed.